### DISCUSSION TITLE
how I add elevation image on vscode

### DISCUSSION THREAD

**Discussion by antoine101009** [netbox-community/devicetype-library#1940] [Q&A] [answered]
Hello

how I can add elevation image on /elevation-images/ ? copy paste paste an URL with VSCODE 

thx to help me ;)

**Comment by danner26** [comment, marked as answer]
Hello, after forking this repo you will need to download the image, name it properly and add it to the elevation images folder. Then you will need to modify the device-types file and make sure either the rear and/or front image definition is true. Then you can commit the changes, push it to your fork, and then make a PR to this repo
